Modules/_decimal/_decimal.c

```diff
@@ -4105,32 +4133,35 @@ PyDec_AsFloat(PyObject *dec)
     return f;
 }
 
+/*[clinic input]
+_decimal.Decimal.__round__
+
+    ndigits: object(c_default="NULL") = None
```

Comment:
Should be positional-only and None is not a valid value.
```suggestion
    ndigits: object = NULL
    /
```

Reply:
> Should be positional-only

Agreed.

> None is not a valid value.

`int.__round__()` supports it.  Maybe `Decimal.__round__()` should too?

Reply:
Conversions to AC should avoid making behavioural changes, we can do that in follow-up PRs.

A